frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/attached-devices.scss

```
@@ -25,16 +25,21 @@
   padding-top: var(--pf-global--spacer--md);
 }
 
-.ceph-ocs-install__create-sc-form {
-  width: 80%;
-}
-
 .ceph-ocs-install__form-wrapper {
-  display: flex;
   margin-top: var(--pf-global--spacer--lg);
   justify-content: space-between;
 }
 
+.ceph-ocs-install__donut-chart {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.ceph-ocs-install__stats__align-center {
```

Follow BEM conventions. See [this](https://en.bem.info/methodology/quick-start/)
```suggestion
.ceph-ocs-install__stats--text-align {
```